pkgs/applications/video/obs-studio/plugins/obs-hyperion/default.nix

```diff
@@ -1,20 +1,26 @@
-{ stdenv, lib, fetchFromGitLab, meson, ninja, pkg-config, obs-studio, libGL
+{ stdenv, lib, fetchFromGitHub, cmake, pkg-config, obs-studio, libGL
 , qtbase }:
 
 stdenv.mkDerivation rec {
   pname = "obs-hyperion";
   version = "1.0.1";
 
-  src = fetchFromGitLab {
+  src = fetchFromGitHub {
     owner = "hyperion-project";
     repo = "hyperion-obs-plugin";
-    rev = "v${version}";
-    sha256 = "sha256-Si+TGYWpNPtUUFT+M571lCYslPyeYX92MdYV2EGgcyQ=";
+    rev = version;
+    sha256 = "sha256-pfWfJWuIoa+74u5J76/GE+OuHkksbwOAPfsR9OGX3L4=";
```

Comment:
Requires submodule see https://github.com/hyperion-project/hyperion-obs-plugin/issues/7

Preferred way would be to symlink our flatbuffers.

Reply:
Thanks